app/src/stores/settings.ts

```diff
@@ -5,6 +5,7 @@ import { unexpectedError } from '@/utils/unexpected-error';
 import { merge } from 'lodash';
 import { defineStore } from 'pinia';
 import { Settings } from '@directus/shared/types';
+import { useUserStore } from '.';
```

Comment:
Lets use explicit path for consistency 

```suggestion
import { useUserStore } from './user';
```